### PR TITLE
Add Sentry support

### PR DESCRIPTION
**Pull request by ryandoherty**
Adds basic support for sending crash logs to Sentry.

If a .sentry config file exists locally, it will create a client and send all uncaught exceptions to it.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -272,6 +281,7 @@ def on_stop(self):
         self._telemetry_connection.telemetry_enabled = False
 
     def showMainView(self, view_name):
+        error = 1/0
```
Comment (brentpicasso) [inline main.py]: should probably remove this.

Comment (ryandoherty) [inline main.py]: LOL oops

File: main.py [hunk 2]
```diff
@@ -486,4 +496,12 @@ def _on_preferences_change(self, menu, config, section, key, value):
                 self._telemetry_connection.telemetry_enabled = False
 
 if __name__ == '__main__':
-    RaceCaptureApp().run()
+    try:
+        RaceCaptureApp().run()
```
Comment (brentpicasso) [inline main.py]: I was going to research if we should re-run the app upon crash to prevent it from exiting completely. I think maybe there are some kivy docs that talk about that.

Comment (ryandoherty) [inline main.py]: Let's do that in a separate PR, this is only for sending crash reports.

Comment (brentpicasso) [inline main.py]: Agreed. Looks good. Shipit

Brent Picasso
Autosport Labs
Technology for Race and Street
